packages/mgt-components/src/components/mgt-login/mgt-login.ts

```diff
@@ -575,16 +543,13 @@ export class MgtLogin extends MgtTemplatedComponent {
                           }
                         }}
                         .personDetails=${details ? JSON.parse(details) : null}
-                        .fallbackDetails=${{ displayName: account.name, mail: account.mail }}
+                        .fallbackDetails=${{ displayName: account?.name, mail: account?.mail }}
```

Comment:
not entirely sure why this needs to be option chained here? 
```suggestion
                        .fallbackDetails=${{ displayName: account.name, mail: account.mail }}
```